tests/visual/label-angular.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Label (Angular)</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    </style>
+</head>
+<body>
+
+    <div id="test-area" class="grid">
+
+        <span>Checkbox</span>
+        <span>Radio</span>
+        <span dir="rtl">Checkbox (RTL)</span>
+        <span dir="rtl">Radio (RTL)</span>
+
+        <section>
+            <input type="checkbox" class="k-checkbox" checked>
+            <kendo-label class="k-checkbox-label">
+                <label class="k-label">Checked</label>
+            </kendo-label>
+        </section>
+
+        <section>
+            <input type="radio" class="k-radio" checked>
+            <kendo-label class="k-radio-label">
+                <label class="k-label">Checked</label>
+            </kendo-label>
+        </section>
+
+        <section dir="rtl">
+            <input type="checkbox" class="k-checkbox" checked>
+            <kendo-label class="k-checkbox-label" dir="rtl">
```

Comment:
you don't need `dir="rtl"` on `kendo-label`. The parent has it.

Reply:
I will keep the `dir="rtl"` on `kendo-label` so we can mimic the exact component rendering.